runtime/Cargo.toml

```diff
@@ -0,0 +1,80 @@
+[package]
+name = "fil_actors_runtime"
+description = "System actors for the Filecoin protocol"
+version = "9.0.0-alpha.1"
+license = "MIT OR Apache-2.0"
+authors = ["ChainSafe Systems <info@chainsafe.io>", "Protocol Labs", "Filecoin Core Devs"]
+edition = "2021"
+repository = "https://github.com/filecoin-project/builtin-actors"
+
+[dependencies]
+fvm_ipld_hamt = "0.5.1"
+fvm_ipld_amt = { version = "0.4.2", features = ["go-interop"] }
+fvm_shared = { version = "0.8.0", default-features = false }
+num-traits = "0.2.14"
+num-derive = "0.3.3"
+serde = { version = "1.0.136", features = ["derive"] }
+lazy_static = "1.4.0"
+unsigned-varint = "0.7.1"
+integer-encoding = { version = "3.0.3", default-features = false }
+byteorder = "1.4.3"
+cid = { version = "0.8.3", default-features = false, features = ["serde-codec"] }
+base64 = "0.13.0"
+log = "0.4.14"
+indexmap = { version = "1.8.0", features = ["serde-1"] }
+thiserror = "1.0.30"
+# enforce wasm compat
+getrandom = { version = "0.2.5", features = ["js"] }
+hex = { version = "0.4.3", optional = true }
+anyhow = "1.0.56"
+fvm_sdk = { version = "1.0.0-rc.3", optional = true }
+blake2b_simd = "1.0"
+fvm_ipld_blockstore = "0.1.1"
+fvm_ipld_encoding = "0.2.2"
+multihash = { version = "0.16.1", default-features = false }
+rand = "0.8.5"
+serde_repr = "0.1.8"
+regex = "1"
+itertools = "0.10"
+
+[dependencies.sha2]
+version = "0.10"
+
+[dev-dependencies]
+derive_builder = "0.10.2"
+hex = "0.4.3"
+
+[features]
```

Comment:
We can remove these variables, they are storage-specific and I don't think we'll need them.